src/mlpack/methods/random_forest/random_forest.hpp

```diff
@@ -409,6 +410,38 @@ class RandomForest
   double avgGain;
 };
 
+/**
+ * Convenience typedef for Extra Trees. (Extremely Randomised Trees Forest)
```

Comment:
```suggestion
 * Convenience typedef for Extra Trees. (Extremely Randomized Trees Forest).
```

Missing typedef and use commonly used name.

Reply:
Sorry, I didn't exactly follow this comment. There is actually a `using` directive below this comment section where I am aliasing `ExtraTrees`. Also, can you please elaborate "commonly used name". Which name are you talking about here?

Reply:
I think @zoq meant that we should use the more commonly used name ("Randomized" vs. "Randomised").

Reply:
Thanks for the clarification! I have addressed this now.

Reply:
Yes, that's what I meant.